packages/amplify-console-integration/src/util.ts

```diff
@@ -0,0 +1,42 @@
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import * as moment from 'moment';
+
+export function createNewProjectDir(mark?: string): string {
+  const timeStamp = moment().format('YYYYMMDDHHmmss');
+  const testProjectDirName = mark ? `console-integ-${mark}-${timeStamp}` : `console-integ-${timeStamp}`;
+  const projectDirpath = path.normalize(path.join(__dirname, '../../..', testProjectDirName));
+  fs.mkdirSync(projectDirpath);
+  return projectDirpath;
+}
+
+export function deleteProjectDir(projectDirpath: string) {
+  return fs.removeSync(projectDirpath);
+}
+
+export function getCLIPath() {
+  if (isCI()) {
+    return 'amplify';
+  }
+  return path.normalize(path.join(__dirname, '../../amplify-cli/bin/amplify'));
+}
+
+export function getProfileName() {
+  return 'amplify-console-integration-test';
+}
+
+export function isCI(): Boolean {
+  return process.env.CI ? true : false;
+}
+
+function stripBOM(content: string) {
```

Comment:
@UnleashedMind can you add a comment here mentioning what the char you're stripping is?

When I try String.fromCharCode(0xfeff), I get an empty string.

Reply:
This is a well-known char, it's called BOM char (hince the name), some OS add it to a text files, and it causes trouble for JSON parser. that's why we need to strip it off before feed the string to JSON.parse.

Reply:
Got it, thanks for clarifying